reactioncommerce/reaction
# 2309 Completed Order Should Break Out Items by Shops

Resolves #2309 

**Testing**
- Clone this Branch and run Reaction with `reaction reset -n && reaction`
- Login with `admin` credentials
- Register and Login as a user in another browser
- Go to the user's profile and select `Become A Seller`
- Go to the browser where the `admin` is logged in and grant the new user permission to create product
- Create product with the user and publish
- Using any Logged on User (i.e `Admin` or `Seller`) of your choice and add at least one product from the new seller shop and default shop (`Reaction`) to your `Cart`
- Checkout items in your `Cart`
- Observe the new design on the `OrderCompleted` page and how items are displayed by grouping them by shops
